cmd/minikube/cmd/status.go

```diff
@@ -35,13 +38,20 @@ import (
 )
 
 var statusFormat string
+var output string
+
+type KubeconfigStatus struct {
+	Correct bool
+	IP      string
+}
 
 // Status represents the status
 type Status struct {
-	Host       string
-	Kubelet    string
-	APIServer  string
-	Kubeconfig string
+	Host             string
+	Kubelet          string
+	APIServer        string
+	Kubeconfig       string
+	KubeconfigStatus KubeconfigStatus
```

Comment:
The `Status.KubeconfigStatus` name stutters. Is there a different naming convention that we could use?

Also, it isn't outwardly apparent what the difference is between `Status.Kubeconfig` and `Status.KubeconfigStatus` is.